internal/infrastructure/service/impl/metrics.go

```diff
@@ -79,19 +88,22 @@ func (srv *metricsService) Start() error {
 	go func() {
 		defer srv.waitGroup.Done()
 		srvAddress := fmt.Sprintf(":%d", srv.config.Metrics.Port)
-		log.Debug().Msgf("metrics: srvAddress=%s", srvAddress)
+		slog.Debug("metrics", slog.String("srvAddress", srvAddress))
 		if err := srv.echo.Start(srvAddress); err != nil && err != http.ErrServerClosed {
-			log.Error().Err(err).Msg("Failed to start metricsService")
+			slog.Error("Failed to start metricsService")
```

Comment:
You are not passing `err` to `slog` here.

Reply:
I added it.

Reply:
The current state include the change to print the error string; resolving this conversation.